### PR TITLE
[SU-47] Run ESLint on code outside of src

### PR DESCRIPTION
**Pull request by nawatts**
Currently, in CI, we only run ESLint on the contents of the `src` directory. This change runs ESLint on all our code (a `.eslintignore` file is added to exclude node_modules, yarn, and minified files).

### CODE REVIEW

File: integration-tests/tests/billing-projects.js [hunk 1]
```diff
@@ -35,19 +34,19 @@ const setAjaxMockValues = async (testPage, ownedBillingProjectName, spendCost) =
     window.ajaxOverridesStore.set([
       {
         filter: { url: /api\/billing\/v2$/ },
-        fn: () => async () => { return new Response(JSON.stringify(projectListResult), { status: 200 }) }
+        fn: () => () => Promise.resolve(new Response(JSON.stringify(projectListResult), { status: 200 }))
```
Comment (cahrens) [inline integration-tests/tests/billing-projects.js:37]: It complains about async? I thought I followed our usual pattern.

Comment (nawatts) [reply integration-tests/tests/billing-projects.js:37]: It complained about `async` functions with no `await`.

Comment (cahrens) [reply integration-tests/tests/billing-projects.js:37]: I see, thanks!

**Comment by cahrens** [comment]
The code LGTM. We should have someone make sure that Jupyter (with Python extension) is working properly.

**Comment by kyuksel** [comment]
The code changes LGTM and I was able to run a Jupyter notebook without issues.
Thank you for taking the time to make this improvement 🙏🏼

**Review by kyuksel — APPROVED** [review]

**Comment by kyuksel** [comment]
You may be asked to reference a Jira ticket for compliance reasons.

**Review by cahrens — APPROVED** [review]

**Comment by cahrens** [comment]
> You may be asked to reference a Jira ticket for compliance reasons.

Thanks for the reminder, Kaan! Yes, you should create a ticket for this and link it to the PR.

**Comment by nawatts** [comment]
Created [SU-47](https://broadworkbench.atlassian.net/browse/SU-47).

**Pull request merged**